templates/miq-template-ext-db.yaml

```diff
@@ -473,6 +477,49 @@ objects:
   spec:
     dockerImageRepository: "${HTTPD_IMG_NAME}"
 - apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: "${HTTPD_SERVICE_NAME}-configs"
+  data:
+    redirects.conf: |
+      # The following redirects files must be included to handle most specific to least specific URLs.
```

Comment:
I think we should hold off on this until https://github.com/ManageIQ/manageiq-pods/pull/197 is merged.  It will significantly simplify the httpd config.